internal/logic/logic.go

```diff
@@ -221,7 +219,6 @@ func (l userMgmtSvcLogic) Activate(id any) *respModel.Response {
 }
 
 func (l userMgmtSvcLogic) UserData(id any) *respModel.Response {
-
 	i, ok := id.(string)
```

Comment:
This is not required as the id will always be a `string`.

Reply:
done